pyproject.toml

```diff
@@ -0,0 +1,18 @@
+[tool.poetry]
+name = "cytocherrypick"
+version = "0.1.0"
+description = "CytoCherryPick is an open source software that provides a set of tools for the selection of representative images from a set of images."
+authors = ["Mike Lippincott <michael.lippincott@cucanschutz.edu>"]
+license = "BSD 3"
+readme = "README.md"
+
+[tool.poetry.dependencies]
+python = "^3.10"
+pandas = "<2.0"
+numpy = "<2.0"
+seaborn = "<2.0"
```

Comment:
I have not started using numpy or pandas 2.0 yet ans there are big differences in the APIs so this is why I specified < 2.0. But I can change down the road?

Reply:
@MikeLippincott - Thank you for the reply - very much up to you. Additional thoughts here: because of the fairly high rate of change in software it's sometimes helpful to "learn as we go" when starting a new project with newer dependency versions or techniques. This gives us a chance to try and keep up with the winds of change. From a software gardening perspective this sometimes yields symbiotic cross-project results; we end up applying what we learn in new ways from garden to garden. I think there's great learning going on here already too; and you're right about the possibility to change down the road at any time.